resolvers.ts

```diff
@@ -586,9 +602,27 @@ export const resolvers: Resolvers = {
         };
       });
     },
-    fedSequencingReads: async (root, args, context) => {
+    fedSequencingReads: async (root, args, context: any) => {
       const input = args.input;
 
+      // NEXT GEN:
+      const nextGenEnabled = await shouldReadFromNextGen(context);
+      if (nextGenEnabled) {
+        const response = await fetchFromNextGen({
+          customQuery: convertSequencingReadsQuery(context.params.query),
```

Comment:
Yeah - I think this is the way to do it! Individual query parsers not shared. I will change the get function asap.